.github/workflows/tests-datapath-verifier.yaml

```diff
@@ -172,8 +172,10 @@ jobs:
         include:
```

Comment:
If 5.10 and netnext are the same why are we testing both?

Reply:
5.10 and netnext are testing the same datapath configuration, but not on the same kernels. It's possible e.g. for the `netnext` job to hit a complexity issue on the latest kernel with that configuration while the `510` job is running fine with the same configuration.

Reply:
If anyone on the datapath side thought I made Linux net-next to be the same as Linux 5.10, I'm going to be very worried for them. I'll change.